src/components/structures/TimelinePanel.tsx

```diff
@@ -1349,11 +1350,11 @@ class TimelinePanel extends React.Component<IProps, IState> {
             if (eventId) {
                 debuglog(
                     "TimelinePanel scrolling to eventId " +
-                        eventId +
-                        " at position " +
-                        offsetBase * 100 +
-                        "% + " +
-                        pixelOffset,
+                    eventId +
```

Comment:
Normally, the code is already formatted by Prettier. Please do not change the formatting :)